core/unix/src/TUnixSystem.cxx

```diff
@@ -568,6 +573,94 @@ TUnixSystem::~TUnixSystem()
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Initialize Unix system interface.
+extern "C"
+{
+   static int FullWrite(int fd, const char *text)
```

Comment:
FullWrite -> Could you tweak the name to be more specific of why it is different from other writing routines?